.vuepress/public/_redirects

```diff
@@ -77,5 +77,5 @@
 
 # Group filters and facets content
 /reference/api/attributes_for_faceting.html         /reference/api/filterable_attributes.html
-/reference/feature/filtering.html                   /reference/api/filtering_and_faceted_search.html
-/reference/feature/faceted_search.html              /reference/api/filtering_and_faceted_search.html
+/reference/features/filtering.html                   /reference/features/filtering_and_faceted_search.html
```

Comment:
Is this url supposed to link to https://docs.meilisearch.com/reference/features/filtering.html, because it lead to a  404.

Reply:
Links on the errors page are hard-coded, so they link to the live docs. I only updated the redirects as a quick patch to catch all instances of the wrong link, so the 404 is expected.

That said, it's probably a good idea to update the errors link directly as well. I've done that in my latest commit to this PR.